README.md

```diff
@@ -9,180 +9,191 @@ SolidLab Research's Challenge [84](https://github.com/solidLabResearch/challenge
-```
-
-10. Please delete the folder `node_modules\@treecg\versionawareldesinldp\node_modules\@inrupt`.
-This is due to conflicting dependencies and
-should be resolved once the `versionawareldesinldp` package has been refactored.
+   Please delete the folder `node_modules\@treecg\versionawareldesinldp\node_modules\@inrupt`.
+   This is due to conflicting dependencies and
+   should be resolved once the `versionawareldesinldp` package has been refactored.
 
 ### Starting the webapp to load
 
-The webapp is a simple vue app that allows you to load the data into the solid pods. 
-
-11. To start the webapp, run the following command from the root of the repository.
-
-```bash
-cd webapp && npm install && npm run dev
-```
-
-12. Open the webapp, select a dataset to be loaded. Click on, `Load selected dataset`.
-
-13. Click on `Get Observation Subjects` to get the observation subjects from the dataset.
-
-14. Then click on, 'Sort observation subjects' and then `submit next observation`.
-Press this button 3 times, till you see the replayer count as 3.
-Now the pod will have 3 observations.
+The webapp is a simple vue app that allows you to load the data into the Solid pods. 
 
-15. Then click on `Submit remaining observations` to submit the rest.
+1. To start the webapp, run the following command from the root of the repository.
 
-The data will be loaded up in the solid pod.
+   ```bash
+   cd webapp && npm install && npm run dev
+   ```
 
-### Aggregation over the solid pods.
+2. Open the webapp, select a dataset to be loaded. Click on the `Load selected dataset` button.
 
-Spun up a new instance of the terminal.
+3. Click on `Get Observation Subjects` button to get the observation subjects from the dataset.
 
-16. Build the project by running the following command from the root of the repository.
+4. Click on the `Sort observation subjects` button.
+5. Click on the `submit next observation`  button 3 times, till you see the replayer count as 3.
+   Now the pod will have 3 observations.
 
-```bash
-npm run build
-```
+6. Click on the `Submit remaining observations` button to submit the rest. 
+   The data will be loaded up in the Solid pod.
 
-17. To start the solid stream aggregator, run the following command from the root of the repository.
+### Aggregation over the Solid pods
 
-```bash
-npm run start aggregation
-```
+Open a new instance of the terminal.
 
-The solid stream aggregator spuns up an HTTP express server at http://localhost:8080/.
+1. Build the project by running the following command from the root of the repository.
 
-The server exposes the following endpoint for the demonstration,
+   ```bash
+   npm run build
+   ```
 
-- /averageHRPatient1
+2. To start the Solid stream aggregator, run the following command from the root of the repository.
 
-For example, the query for passed when requesting the endpoint /averageHRPatient1 is,
+   ```bash
+   npm run start aggregation
```

Comment:
as the @solid/community-server is not in the dependancy list because of the issue we faced before. I will add it here, as an extra step to install before for the aggregator service to start. Rest, looks good to me.